### PR TITLE
Implement constant-luminance HLG OOTF

### PR DESCRIPTION
**Pull request by sekrit-twc**
Addresses #71.

### CODE REVIEW

**Comment by sekrit-twc** [comment]
@dzung-hoang @haasn RFC

File: src/zimg/colorspace/operation_impl.cpp [hunk 1]
```diff
@@ -69,6 +69,78 @@ class GammaOperationC final : public Operation {
...
+	{
+		const float gamma = 1.2f;
+
+		for (unsigned i = left; i < right; ++i) {
+			float r = src[0][i] * m_scale;
+			float g = src[1][i] * m_scale;
+			float b = src[2][i] * m_scale;
+
+			float yd = std::max(m_kr * r + m_kg * g + m_kb * b, FLT_MIN);
+			float ys = zimg_x_powf(yd, (gamma - 1.0f) / gamma);
```
Comment (rusxg) [inline src/zimg/colorspace/operation_impl.cpp]: Maybe I am missing something, but I can't find where is the multiplication by 'alpha' parameter, given in BT.2100 as `(Lw - Lb)/12^gamma`. For Lw = 1000, Lb = 0, it's about 50.69 (far from 1). Should it be in m_scale somehow?

Same thing for AribB67InverseOperationC

Comment (sekrit-twc) [inline src/zimg/colorspace/operation_impl.cpp]: z.lib uses 0-1 range for gamma-encoded domain, which makes the 'alpha' factor unity according to BT.2100.

**Comment by sekrit-twc** [comment]
I tested by converting "LG_Chess_HDR-noaudio.mp4" to HLG. Nothing looks obviously wrong, and the constant-luminance method shows some improvement in color saturation.

**Pull request merged**